src/lj_dispatch.c

```diff
@@ -66,6 +78,9 @@ void lj_dispatch_init(GG_State *GG)
 #if LJ_TARGET_MIPS
   memcpy(GG->got, dispatch_got, LJ_GOT__MAX*4);
 #endif
+#if LJ_TARGET_PPC
```

Comment:
Just add it as an `||` to the above condition?

Reply:
Hi, I think probably the right thing is to make like `LJ_USE_OWN_GOT` or something and have that enabled per target since otherwise it can be hard to know what is MIPS/PPC specific and what's used for this GOT feature. I'll try to find some time in the next while to make an update! thx for commenting